src/main/java/core/org/akaza/openclinica/service/crfdata/EnketoUrlService.java

```diff
@@ -205,6 +205,7 @@ public FormUrlObject getActionUrl(String subjectContextKey, PFormCacheSubjectCon
         if(eventCrf==null){
             UserAccount userAccount = userAccountDao.findByUserId(ub.getId());
             eventCrf= createEventCrf(formLayout,studyEvent,subject,userAccount);
+            logger.info("creating new event crf {}",eventCrf.getEventCrfId());
```

Comment:
I think we should print event and crf id too, so we know why they don't match up.